docs/pages/jpscore/jpscore_trains.md

```diff
@@ -5,189 +5,133 @@ tags: [jpscore, file, simulation]
+<train_type>
+    <train type="RE" agents_max="15" length="9">
+        <door id="1" distance="1" width="1"/>
+        <door id="3" distance="5" width="1"/>
+    </train>
 
-```
- <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
- <train_type>
-  <train type="RE" agents_max="10" length="5">
-   <door id="1">
-      <vertex px="0.5" py="0.0" />
-      <vertex px="2" py="0.0" />
-    </door>
-    <door id="2" outflow="3">
-      <vertex px="4" py="0.0" />
-      <vertex px="5" py="0.0" />
-    </door>
-  </train>
-  <train type="ICE" agents_max="800" length="4">
-    <door id="1">
-      <vertex px="1" py="0" />
-      <vertex px="2" py="0" />
-    </door>
-    <door id="2">
-      <vertex px="4" py="0" />
-      <vertex px="5" py="0" />
-    </door>
+    <train type="ICE" agents_max="20" length="23">
+        <door id="1" distance="1.5" width="2"/>
+        <door id="2" distance="10" width="3"/>
+        <door id="3" distance="18" width="2"/>
     </train>
- </train_type>
+</train_type>
 ```
+
+![Schematic overview of train type definition.]({{ site.baseurl }}/images/trains/traintype.png)
+
 ### Capacity of a train
 
 The number of agents in a train is calculated every time step as the sum of
 all agents passing through the train's doors.
 
 When this number exceeds the `agents_max` parameter, all train's doors are closed.
 
-## Transformation of coordinates
-
-While train's coordinates are **relative** to the start if the track (`track_start`), 
-Door's coordinates are **relative** to the point train's start (`train_start`) as defined in the [Train timetable](#train-timetable) file.
-
-![Coordinate systems of trains.]({{ site.baseurl }}/images/coordinates.png)
-
-Therefore, the global coordinates of the train's door `[A,B]` is calculated as follows
-
-$$
-A_{global} = track\_start + train\_start + A_{local},
-$$
-
-$$
-B_{global} = track\_start + train\_start + B_{local}.
-$$
-
-
-## Algorithm
-
-Given a `platform` with at least one `track`, the geometry will be changed every time a train arrives and leaves the platform.
-
-This will be done by projecting the train's doors on the track. With "projection" we mean along the orthogonal direction to the door.
-
-![platform]({{ site.baseurl }}/images/platform.png)
-
-Every projection point is mapped to the corresponding wall on the track.
+## Train timetable
+### Definition 
+A train is defined through the following information: 
+- `id` (int): id of the train
+- `type` (string): identifier of the train defined with the train types
+- `track_id` (int): id of the track the train arrives
+- `train_offset` (float): offset of the train to the track start
+- `reversed` (bool): false: train starts at track start, true: train starts at track end, default: false
+- `arrival_time` (float): time the train arrives
+- `departure_time` (float): time the train departs
+
+{%include note.html content="The parameter `reversed` add doors starting from track end in direction of track start!."%}
```

Comment:
{%include note.html content="The parameter `reversed` adds doors starting from track end in direction of track start!"%}

Reply:
Done